packages/runtime/src/components/builtin/Slot/SlotRoot.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import { ReactNode, useContext } from "react"
+import { PropsContext } from "../../../next/components/MakeswiftSlot"
+
+type Props = {
+  children: ReactNode
+  showFallback: boolean
+}
+
+export const SlotRoot = (props: Props) => {
+  const ctx = useContext(PropsContext)
+
+  return (
+      <>
+        { props.showFallback ? props.children : ctx.fallback}
```

Comment:
There's some extraneous spaces here. Do you have Prettier configured to run on save on your IDE?

Reply:
Working for me in cosmos but not our public repo, and I don't know why. I haven't overridden any config or anything.

Reply:
Are you working at the top-level of the monorepo?

Reply:
Yeah I'm at the top level. From what I'm seeing when I run prettier using the CLI, I'm wondering if others have been having this issue too. Running prettier even just for`packages/runtime/src` gives quite a few changes
<img width="792" alt="image" src="https://github.com/user-attachments/assets/1538304c-6fc0-4eae-af32-bb1ba97f0486" />

Odd. Anyway I'll take care of formatting the relevant files here using the CLI

Reply:
for me, later: `pnpm prettier packages/runtime/src --write`

Reply:
This is why we should enforce it in CI, hahaha